src/inspection.ts

```diff
@@ -228,6 +228,14 @@ function checkBody(
     return errors;
   }
 
+  // Minimum character body length
+  if (inputs.minBodyLength && bodyLines.join().length < inputs.minBodyLength) {
```

Comment:
Should it could newlines as characters?

The lines are split by `\n`, so they are not included in `bodyLines`. But calling `join()` without any arguments adds a `,` between each line, effectively counting what was originally the line breaks.

I'd prefer `join('')` to not count those, but it's definitely not blocking

Reply:
I think both counting and not counting the newlines are equally fine as those are mostly corner cases.
But given how unintuitive it might be then counting would be better. So count newlines the same as counting whitespaces